pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -664,7 +667,7 @@ func (k *Kubernetes) Deploy(komposeObject kobject.KomposeObject, opt kobject.Con
 			if err != nil {
 				return err
 			}
-			log.Infof("Successfully created PersistentVolumeClaim: %s", t.Name)
+			log.Infof("Successfully created PersistentVolumeClaim: %s of size %s. PVC's will not work unless a PersistentVolume has been created.", t.Name, PVCRequestSize)
```

Comment:
@kadel, I had written something like this, but removed as @cdrage said, https://github.com/kubernetes-incubator/kompose/pull/519#issuecomment-290400229